src/main/java/frc/robot/commands/Turret/TurretCommand.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.Turret;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants;
+import frc.robot.subsystems.IntakeSubsystem;
+import frc.robot.subsystems.TurretSubsystem;
+
+/** An example command that uses an example subsystem. */
+public class TurretCommand extends Command {
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+  private final TurretSubsystem turretSubsystem;
+
+  /**
+   * Creates a new ExampleCommand.
+   *
+   * @param subsystem The subsystem used by this command.
+   */
+  public TurretCommand(final TurretSubsystem turretSubsystem) {
+    this.turretSubsystem = turretSubsystem;
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(turretSubsystem);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    turretSubsystem.setTurretVelo(1);
```

Comment:
Can we make this into a command factory in the turret subsystem?  There is a lot of template code that we could remove for a more readable solution.  

A Command Factory means that a method in the turret subsystem will return a command that can be used to assign to a trigger or button